tox.ini

```diff
@@ -15,6 +15,10 @@ deps =
     bandit
 
 commands =
+    # Need master version of securesystemslib till this commit is in a release:
+    # https://github.com/secure-systems-lab/securesystemslib/commit/ba5f52118d308d2bc67ad11b5947a4ea7ef8f2ed
```

Comment:
I released v0.11.2, which contains the commits needed by this pull request.  Can you update this line to use v.0.11.2?  Let me know if you encounter any issues.
https://github.com/secure-systems-lab/securesystemslib/releases/tag/sslibv0.11.2

Reply:
A default installation of v0.11.2 only supports ed25519 in pure python.  To enable rsa, ecdsa, c extensions, etc.,  you need to install the optional dependencies...

```
pip install securesystemslib[crypto]
pip install securesystemslib[pynacl]
```

Reply:
@yashsriv You don't have to worry about getting `in-toto` to work in pure Python.  For now, you just have to update the requirements.txt and setup.py files to include the `crypto` and `pynacl`.

In setup.py, it's something like:
```
setup(
    ...
    install_requires=["securesystemslib[crypto]"],
    ...
)
```

In requirements.txt, it's something like:

```
securesystemslib[security] == 0.11.2 
```